### PR TITLE
Feature/local 3w

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/root/views/Country/ThreeW/projectTableColumns.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    'sector',
+    'Sector',
+    (item) => item.primary_sector_display,
+  ),
+  createNumberColumn<Project, string | number>(
+    'budget',
+    'Total Budget',
+    (item) => item.budget_amount,
+    undefined,
+    { normal: true }
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/projectTableColumns.ts:26]:
```suggestion
    { normal: true },
```

File: src/root/views/Country/ThreeW/projectTableColumns.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    'disasterType',
+    'Disaster Type',
+    (item) => item.dtype_detail?.name,
+  ),
+  createNumberColumn<Project, string | number>(
+    'peopleTargeted',
+    'People Targeted',
+    (item) => item.target_total,
+    undefined,
+    { normal: true }
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/projectTableColumns.ts:43]:
```suggestion
    { normal: true },
```

File: src/root/views/Country/ThreeW/projectTableColumns.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  ),
+  createNumberColumn<Project, string | number>(
+    'peopleReached',
+    'People Reached',
+    (item) => item.reached_total,
+    undefined,
+    { normal: true }
```
Comment (AdityaKhatri) [inline src/root/views/Country/ThreeW/projectTableColumns.ts:50]:
```suggestion
    { normal: true },
```

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: src/root/components/Card/index.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  className?: string;
+  children?: React.ReactNode;
```
[the remark this answers is not in the document]
Comment (frozenhelium) [reply src/root/components/Card/index.tsx:8]: Children can be optional

File: src/root/components/KeyFigureCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import { _cs } from '@togglecorp/fujs';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  className?: string;
+  children?: React.ReactNode;
+}
+
+function KeyFigureCard(props: Props) {
```
[the remark this answers is not in the document]
Comment (frozenhelium) [reply src/root/components/KeyFigureCard/index.tsx]: Removed

File: src/root/utils/restRequest/index.ts [hunk 6]
```diff
@@ -3,3 +3,9 @@ import useRequest from './useRequest';
 import useLazyRequest from './useLazyRequest';
 
 export { RequestContext, useRequest, useLazyRequest };
+
+export type ListResponse<T> = {
+  count: number;
+  results: T[];
+  next?: string;
+};
```
[the remark this answers is not in the document]
Comment (frozenhelium) [reply src/root/utils/restRequest/index.ts:11]: That's exactly why I put it in the `index.ts`

**Pull request merged**